.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: ruby
+rvm:
+  - "1.9.3-p194"
```

Comment:
Not a huge fan of testing on such an old version of ruby with known security flaws...